docs/_tutorials/getting_started.md

```diff
@@ -13,3 +13,321 @@ redirect_from:
+                },
+                "accessory": {
+                    "type": "button",
+                    "text": {
+                        "type": "plain_text",
+                        "text": "Click Me"
+                    },
+                    "action_id": "button_click"
+                }
+            }
+        ],
+        text=f"Hey there <@{message['user']}>!"
+    )
+
+# Start your app
+if __name__ == "__main__":
+    app.start(port=int(os.environ.get("PORT", 3000)))
+```
+
+<!-- TODO - Review my usage of "List" (Array?) and clarification of the role of `text` -->
+The values passed to `say()` are now a list of `blocks` and a string of `text`. Blocks are the building components of a Slack message and can range from text to images to datepickers. In this case, your app will respond with a section block that includes a button as an accessory. Since we're using `blocks`, the `text` is a fallback string to display in notifications.
+
+You'll notice in the button `accessory` object, there is an `action_id`. This will act as a unique identifier for the button so your app can specify what action it wants to respond to.
+
+> 💡 The [Block Kit Builder](https://api.slack.com/tools/block-kit-builder) is an simple way to prototype your interactive messages. The builder lets you (or anyone on your team) mockup messages and generates the corresponding JSON that you can paste directly in your app.
+
+Now, if you restart your app and say "hello" in a channel your app is in, you'll see a message with a button. But if you click the button, nothing happens (*yet!*).
+
+Let's add a handler to send a followup message when someone clicks the button:
+
+<!-- TODO - I matched the ordering of the (ack, body, say) with JavaScript. Is this the ideal order? -->
+```python
+import os
+from slack_bolt import App
+
+# Initializes your app with your bot token and signing secret
+app = App(
+    token=os.environ.get("SLACK_BOT_TOKEN"),
+    signing_secret=os.environ.get("SLACK_SIGNING_SECRET")
+)
+
+# Listens to incoming messages that contain "hello"
+@app.message("hello")
+def message_hello(message, say):
+    # say() sends a message to the channel where the event was triggered
+    say(
+        blocks=[
+            {
+                "type": "section",
+                "text": {
+                    "type": "mrkdwn",
+                    "text": f"Hey there <@{message['user']}>!"
+                },
+                "accessory": {
+                    "type": "button",
+                    "text": {
+                        "type": "plain_text",
+                        "text": "Click Me"
+                    },
+                    "action_id": "button_click"
+                }
+            }
+        ],
+        text=f"Hey there <@{message['user']}>!"
+    )
+
+@app.action("button_click")
+def action_button_click(body, ack, say):
+    # Acknowledge the action
+    ack()
+    say(f"<@{body['user']['id']}> clicked the button")
```

Comment:
Nit - I matched the argument order from _Getting Started with Bolt for JavaScript_ (`(ack, body, say)`). Is there is a best practice order that we'd like to encourage? I'm happy to update both Python/JavaScript to match whatever order we think is best.

Reply:
I prefer order in which they are used, or alphabetical, but I think matching what the original docs do is fine. It's also likely what I did for the Basics section (assumption, not looking atm).

Reply:
Thanks for sharing your thoughts, @misscoded. I agree that it makes sense for the Python docs (getting started & concepts) to match the order of the JavaScript docs. Let's stick with that for now!

I do think our docs are a good place to implicitly recommend best practices, such as argument order. But let's save this for a future discussion.